campaign.html

```diff
@@ -0,0 +1,521 @@
+
+                    <li class="nav-item dropdown">
+                        <a class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
+                            aria-expanded="false" href="#">Support</a>
+                        <div class="dropdown-menu">
+                            <a class="dropdown-item" href="https://www.juegacomocampeon.com/#volunteer">Volunteer</a>
+                            <a class="dropdown-item" href="https://www.juegacomocampeon.com/#host-a-drive">Host a
+                                Drive</a>
+                            <a class="dropdown-item" href="https://www.juegacomocampeon.com/#contribute">Contribute</a>
+                            <a class="dropdown-item" href="https://www.juegacomocampeon.com/#sponsor">Sponsor</a>
+                            <a class="dropdown-item" href="https://www.juegacomocampeon.com/#employment">Jobs</a>
+                        </div>
+                    </li>
+
+                    <li class="nav-item"><a class="nav-link" href="https://www.juegacomocampeon.com/#gallery">Media</a>
+                    </li>
+                    <li class="nav-item"><a class="nav-link"
+                            href="https://www.juegacomocampeon.com/#contact">Contact</a></li>
+
+                </ul> <!-- navbar -->
+
+            </div>
+
+        </nav>
+
+    </header>
+
+    <div class="">
+
+        <!-- GRID CAMPAIGN -->
+
+        <section id="campaign" class="campaign text-justify">
+            <h1>GRID CAMPAIGN</h1>
+            <hr />
+            <article>
+                <h3 id="contribute">LET’S BUILD A BASKETBALL COURT !</h3>
+                <p>
+                    By clicking on one of the balls below, you’ll be making a donation in that amount to support Juega Como Campeon (JCC) and positively impact the lives of underprivileged Dominican youth where we stress educational reinforcement, athletic excellence, gratitude, community service, integrity and personal responsibility.
+                    <br>
+                    <br>
+                    We have received the go ahead from the local sports ministry in Puerto Plata that we can build a basketball court on their land to be used daily by our student athletes. We just need the added funds to do it. So please spread the word (and this link) to your groups and invite them to consider JCC to be a part of their philanthropy this Christmas season. 
+                    <br>
+                    <br>
+                    And if you feel inclined to give more than the corresponding amount, please write us at juegacomocampeon@gmail.com                    <br>
+                    <br>
+                    <div style="width: 100%; display: flex; justify-content: center;">
+                    <img src="img/picture gallery 10.png" style="width: 95%; border: 3px gray solid;" alt="Bilingual Sports Academy">
+                    </div>
+                </p>
+            </article>
```

Comment:
Prettier me esta dando este error, lo puedo arreglar si quieres. El problema es que tienes un `<p>` que contiene otros elementos dentro. Deberia ser in `div` que contenga multiples `p`. Tambien abajo del todo hay un `<script>` que no tiene el otro tag.